source/features/easy-toggle-files.tsx

```diff
@@ -9,9 +9,22 @@ function toggleFile(event: DelegateEvent<MouseEvent>): void {
 	const elementClicked = event.target as HTMLElement;
 	const headerBar = event.delegateTarget;
 
-	// The clicked element is either the bar itself or one of its 2 children
-	if (elementClicked === headerBar || elementClicked.parentElement === headerBar) {
-		$('[aria-label="Toggle diff contents"]', headerBar)
+	if (
+		// The clicked element is either the bar itself or one of its 2 children
+		elementClicked === headerBar
+		|| elementClicked.parentElement === headerBar
+		// React
+		|| elementClicked.matches([
+			'[class^="DiffFileHeader-module__diff-file-header"] > div',
+			'[class^="DiffFileHeader-module__diff-file-header"] > div > div',
+		])
```

Comment:
Not a fan of this, maybe we can find better. It looks like all the active elements are either `button` or `a` so we should be able to safely exclude them:

```suggestion
		|| !elementClicked.closest('a, button')
```